how-to/debug-buildfarm-builder.rst

```diff
@@ -26,6 +32,35 @@ To anoint yourself to be part of that team in qastaging, see
 
 3. Click the "Switch to manual-mode" button.
 
+.. note::
+
+   Setting the builder to ``manual`` prevents new builds to be dispatched to
+   the builder, but the container will be removed.
+
+(Alternative) Set a builder to disabled via CLI
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You need to have the ubuntu-archive-tools installed.
+
+.. code-block:: sh
+
+    $ git clone https://git.launchpad.net/ubuntu-archive-tools
+    $ sudo apt install python3-launchpadlib python3-ubuntutools
+
+
+For example, you want to debug the ``lcy02-amd64-004`` instance.
+
+.. code-block:: sh
+
+    ❯ ./manage-builders -l qastaging -b qastaging-lcy02-amd64-004 --disabled
```

Comment:
If the point s to disable the builder, this should be `--disable`.
`--disabled` will just filter the list to only show disabled builders, it won't change the status.

Reply:
Thanks for the catch. I am not super happy about the CLI design, as the `action` commands and the `filter` commands are way too similar.